src/index.ts

```diff
@@ -54,7 +54,7 @@ const stargate = async (req: Request, res: Response) => {
   if (CONFIG.APIGenerated.mainnet === "TRUE") {
       res.send({
         // TODO: dynamic Milkomeda address from server
-        current_address: 'mainnet_address_here', // TODO: no mainnet address for now
+        current_address: 'addr1w8pydstdswmdqmg2rdt59dzql3zgfp9pt8sulnjgalycwdsj9js7w',
```

Comment:
hmm we can get it from the bridge's smart contract actually. We can merge that one. I'll put @bartfish on adding the necessary change to get that.

Reply:
I've added an issue #5 to get that sorted

Reply:
@NicolasDP Agree. I would merge this in the meantime tho